src/Accelerators/NNPA/Transform/ZLow/ZLowRewrite.cpp

```diff
@@ -100,6 +112,330 @@ class StickViewUnstickRemovalPattern : public OpRewritePattern<ZLowStickOp> {
   }
 };
 
+// clang-format off
+///
+/// * Pattern to rewrite
+/// ```
+/// zlow.unstick -> affine.for (affine.load -> affine.store) -> zlow.stick
+///    |                            |
+///    |                            '--------> affine.store) -> zlow.stick
+///    |
+///    '----------> affine.for (affine.load -> affine.store) -> zlow.stick
+///                                                              ^
+/// zlow.unstick -> affine.for (affine.load -> affine.store) ----'
```

Comment:
Can this Op be any Op to operate on the sticky data?

Reply:
In general, yes. The purpose here is to remove zlow.unstick/zlow.stick, so I check zlow.unstick/zlow.stick only.